services/hunt_dispatcher.go

```diff
@@ -106,7 +106,10 @@ type IHuntDispatcher interface {
 		acl_manager vql_subsystem.ACLManager,
 		hunt *api_proto.Hunt) (*api_proto.Hunt, error)
 
-	// Deprecated - use GetHunts for paged access
+	ListActiveHunts(ctx context.Context,
```

Comment:
Since this function is deprecated we probably dont want to add another function like it. We need a better approach